Source/WebCore/css/CSSFontFace.h

```diff
@@ -107,7 +107,7 @@ class CSSFontFace final : public RefCounted<CSSFontFace> {
     RefPtr<CSSValueList> families() const;
     std::span<const UnicodeRange> ranges() const { ASSERT(m_status != Status::Failure); return m_ranges.span(); }
 
-    void setFontSelectionCapabilities(FontSelectionCapabilities capabilities) { m_fontSelectionCapabilities = capabilities; }
+    void setFontSelectionCapabilities(const FontSelectionCapabilities& capabilities) { m_fontSelectionCapabilities = capabilities; }
```

Comment:
This is not a large type, I am not convinced passing by const reference is actually better here?

Reply:
Will remove.